src/main/scala/uk/gov/hmrc/ct/accounts/frs102/boxes/ACQ5021.scala

```diff
@@ -29,7 +29,7 @@ case class ACQ5021(value: Option[Boolean]) extends CtBoxIdentifier(name = "Goodw
     import boxRetriever._
     collectErrors(
 
-      cannotExistIf(ac42.noValue && ac43.noValue),
+      cannotExistIf(hasValue && ac42.noValue && ac43.noValue),
 
       failIf(ac42.hasValue || ac43.hasValue) {
         atLeastOneBoxHasValue("balanche.sheet.intangible.assets", this, acq5022)
```

Comment:
I missed this typo on the last PR ... could you please correct the "balanche" to "balance" ?:)